### ISSUE TITLE
why did you write these?are you ok?

### ISSUE THREAD

**Issue by Jincheng2012** [Ha3MrX/DDos-Attack#132]
print "[                    ] 0% "
time.sleep(5)
print "[=====               ] 25%"
time.sleep(5)
print "[==========          ] 50%"
time.sleep(5)
print "[===============     ] 75%"
time.sleep(5)
print "[====================] 100%"
time.sleep(3)
Why write these codes? They only implemented a fake progress bar, which had no practical effect and slowed down the time.
                                                                                                                                                         ----------By Baidu Translate
The following content is the same as above：
与上面相同：
为什么要写这些代码？他们只实现了一个假的进度条，浪费时间，没有p用。tmd作者你strong不要在GitHub上好吧，脑瘫赶紧去医院治，再说都2022年写的，还tm用python2.x写，咋滴，你暗恋python2啊，老子tm改写python3的时候全给你删了，我连注释都懒得注释，来，随便怼。
<img width="960" alt="diam" src="https://github.com/user-attachments/assets/d582ee76-734a-41f4-912f-399aaf779551">

**Comment by nuqko** [comment]
if you read the full program you can tell this was probably written by a 12 year old, nothing in this script makes sense and this entire thing is painfully inefficient at actually ddosing, there's no implementation for botnet (to be expected though), and it sends packets using socket which will just be ignored by whatever your sending packets to. all in all the script just sucks man

**Comment by Always-come-back** [comment]
@asmetics I agree with you, the program is terrible. No parentheses are used for print. And "raw_input" is used. strange.
